packages/forms/src/components/UniversalFormOverlay/index.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { useRef } from 'react';
+import { Button } from 'react-rainbow-components';
+import InternalOverlay from 'react-rainbow-components/components/InternalOverlay';
+import useOutsideClick from '../../../../hooks/src/useOutsideClick';
+import useUniqueIdentifier from '../../../../hooks/src/useUniqueIdentifier';
```

Comment:
here use import as ```import { useOutsideClick, useUniqueIdentifier } from '@rainbow-modules/hooks';``` and then add `@rainbow-modules/hooks` as dependencies of this module